backend/cinema/serializers.py

```diff
@@ -114,7 +114,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], attrs["seat"], data["movie_session"].cinema_hall, ValidationError
+            attrs["row"],
```

Comment:
The reason why images are not loading is because the media folder is empty

Reply:
Uploaded pictures:
<img width="1416" alt="Screenshot 2023-06-30 at 09 20 14" src="https://github.com/mate-academy/py-cinema-full-stack/assets/121613917/1e3e5eab-e5d2-4f48-9721-e35a0bfdf050">